src/python/pants/backend/codegen/protobuf/scala/scalapbc.py

```diff
@@ -31,12 +52,31 @@ def register_options(cls, register):
                 "A list of addresses to `jvm_artifact` targets for the runtime "
                 "dependencies needed for generated Scala code to work. For example, "
                 "`['3rdparty/jvm:scalapb-runtime']`. These dependencies will "
-                "be automatically added to every `protobuf_sources` target. At the very leasst, "
+                "be automatically added to every `protobuf_sources` target. At the very least, "
                 "this option must be set to a `jvm_artifact` for the "
                 f"`com.thesamet.scalapb:scalapb-runtime_SCALAVER:{cls.default_version}` runtime library."
             ),
         )
+        register(
+            "--jvm-plugin-artifacts",
```

Comment:
Would this option be better named just `--jvm-plugins`?

Reply:
No preference.